contracts/HubPoolInterface.sol

```diff
@@ -26,6 +21,11 @@ interface HubPoolInterface {
         // A positive number indicates that the HubPool owes the SpokePool funds. A negative number indicates that the
         // SpokePool owes the HubPool funds. See the comment above for the dynamics of this and netSendAmounts
         int256[] runningBalances;
+        // Used as the index in the bitmap to track whether this leaf has been executed or not.
+        uint32 leafId;
```

Comment:
Should this be the same size as the `poolRebalanceLeafCount` in the HubPool functions, aka uint8?

Reply:
I think yes! as long as the leafId in the relayer refund remains large we are good.